src/solution.c

```diff
@@ -429,6 +434,56 @@ static void sol_thd_sleep(systime_t *deadline, systime_t interval)
   chSysUnlock();
 }
 
+/** Calculates Iono and Tropo correction
+ * \param n_ready_tdcp The number of measurements written to `m_tdcp`
+ * \param nav_meas_tdcp Pointer to measurements array
+ * \param pos_ecef position in WGS84 Earth Centered, Earth Fixed (ECEF) Cartesian
+ * \param pos_llh position in WGS84 geodetic coordinates*/
+void calc_iono_tropo(u8 n_ready_tdcp,
```

Comment:
Oh, but I see it needs the iono params struct... hmm I guess that would need to be made a parameter :-/

@benjamin0 Do you think it would be useful to have the correction function in libswiftnav?